### ISSUE TITLE
508-defect-3 [COGNITION]: Directions link should not open in a new tab and include "Google Maps"

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#39098]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user or someone with cognitive disabilities, I want to know a link's purpose (why I'm using it) and target (where it will take me too). As a screen magnifier user, I want to know ahead of time if I'll be redirected to a new window so I'll know I'm on a different page.

## Details
Opening links in a new tab without warning users can result in the following issues:

1. Screen reader users, screen magnifier users, and users with certain cognitive impairments can become disoriented when they are taken to a new window
1. The use of the back button, the most used button within the browser, is broken.
1. Certain user agents, like kiosks, are unable to open new windows or tabs.

## Acceptance Criteria

To prevent these issues, provide a clear target (where someone will be redirected too upon clicking the link), and to create consistency with the facility locator-- we recommend the following changes:

- [ ] Directions link opens on the same page
- [ ] "Google Maps" is included in the link text e.g. `Directions (Google Maps)`

## Steps to Recreate

1. Enter `<https://www.va.gov/pittsburgh-health-care/locations/h-john-heinz-iii-department-of-veterans-affairs-medical-center/>` in browser
1. Find "Directions" link and click on it
1. Confirm it opens in a new tab 

## WCAG or Vendor Guidance (optional)

[G200: Opening new windows and tabs from a link only when necessary](https://www.w3.org/TR/WCAG20-TECHS/G200.html)
- “In general, it is better not to open new windows and tabs since they can be disorienting for people, especially people who have difficulty perceiving visual content.”
- Exceptions might include getting contextual help on an  online form or not wanting to break a secure session.

## Screenshots or Trace Logs
<img width="1483" alt="Screen Shot 2022-03-24 at 3 28 16 PM" src="https://user-images.githubusercontent.com/14154792/159995218-3a979cf4-4875-4d32-801b-7f43b64bb178.png">
